### ISSUE TITLE
messages not sending

### ISSUE THREAD

**Issue by vaciveVeracious** [igdmapps/igdm#1117]
every now and then when i press send, it says its sending but it doesnt send

**Comment by scooter691** [comment]
Yes, I have this on almost all messages Answering works but sending is not

**Comment by baptx** [comment]
Are you using a VPN or proxy? I have the same problem since several weeks, it looks like Instagram has changed how they detect spam recently: https://github.com/igdmapps/igdm/issues/1133
